chainer/iterators/multiprocess_iterator.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, dataset, batch_size, repeat=True, shuffle=True,
 
         self._finalized = None
 
+        self._previous_epoch_detail = -1.
```

Comment:
Actually, there is no need to use `None` as the invalid value. I chose `None` because we can easily understand it is invalid. We can use `-1` instead of `None` with enough documents.